README.md

```diff
@@ -1941,6 +1941,51 @@ Usage typically looks like:
     }
 ~~~
 
+##### `kerb`
```

Comment:
After doing the mod_melon one I am thinking this variable *kerb => true|false* should just not exist in fact.